contracts/StablecoinConverter.sol

```diff
@@ -286,7 +288,7 @@ contract StablecoinConverter is EpochTokenLocker {
         uint256 burntFees = uint256(tokenConservation[0]) / 2;
         require(utility.add(burntFees) > disregardedUtility, "Solution must be better than trivial");
         // burntFees ensures direct trades (when available) yield better solutions than longer rings
-        uint256 objectiveValue = utility - disregardedUtility + burntFees;
+        uint256 objectiveValue = utility.add(burntFees).sub(disregardedUtility);
         checkAndOverrideObjectiveValue(objectiveValue);
         grantRewardToSolutionSubmitter(burntFees);
         tokenConservation.checkTokenConservation();
```

Comment:
I would prefer to check the token conservation, before we calculate the burntFees with a possible overflow.
It should not matter as things get anyways reverted, but still, it feels cleaner.

Reply:
Sure thing! No problem!